pkg/cmd/server/origin/master.go

```diff
@@ -494,11 +494,22 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 		handler = contextHandler
 	}
 
+	if c.Options.ServingInfo.MaxRequestsInFlight > 0 {
+		longRunningRE := regexp.MustCompile("[.*\\/watch$][^\\/proxy.*]")
```

Comment:
Hahahhahahaha you think the person who implemented this upstream thought of those.  

> On Jun 6, 2015, at 11:32 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In pkg/cmd/server/origin/master.go:
> 
> > @@ -494,11 +494,22 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
> >  		handler = contextHandler
> >  	}
> >  
> > +	if c.Options.ServingInfo.MaxRequestsInFlight > 0 {
> > +		longRunningRE := regexp.MustCompile("[.*\\/watch$][^\\/proxy.*]")
> what about pods/exec, pods/proxy, pods/logs, etc?
> 
> —
> Reply to this email directly or view it on GitHub.
>